tests/EdgeToEdge/SkinTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Tests\EdgeToEdge;
+
+use Symfony\Component\BrowserKit\Cookie;
+use Symfony\Component\HttpFoundation\Response;
+
+class SkinTest extends WebRouteTestCase {
+
+	public function testDefaultSkinUsed(): void {
+		$client = $this->createClient( $this->getDummyConfig(), null, self::DISABLE_DEBUG );
+		$client->request( 'GET', '/' );
+
+		$this->assertContains( '10h16', $client->getResponse()->getContent() );
+		$this->assertNoSkinResponseCookie( $client->getResponse() );
```

Comment:
If you put the assert methods (and all methods really) right below where they are first used, less vertical scrolling is needed to real the class.